src/codegen.cpp

```diff
@@ -500,7 +500,7 @@ static AttributeList get_donotdelete_func_attrs(LLVMContext &C)
     FnAttrs = FnAttrs.addAttribute(C, Attribute::NoUnwind);
     return AttributeList::get(C,
             FnAttrs,
-            Attributes(C, {Attribute::NonNull}),
+            Attributes(C, {}),
```

Comment:
Can we just use the default constructor of AttributeSet here instead?

Reply:
Probably. I just submitted this as a quick drive-by while I was working on something else. Feel free to make a PR that fixes it better :).